kgitb/__main__.py

```diff
@@ -1,12 +1,17 @@
 import logging
 import sys
 
-from .config import GH_TOKEN, HOST, PORT
-from . import app
+from flask import Flask
+
+from .config import GH_TOKEN, HOST, PORT, HIDDEN
+from . import routes
 
 log = logging.getLogger(__name__)
 
 
+app = routes.setup(Flask(HIDDEN[1:-1]))
+
+
 def main():
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Missing function docstring [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AC0111)